mlflow/utils/async_logging/run_batch.py

```diff
@@ -9,27 +9,28 @@
 class RunBatch:
     def __init__(
         self,
-        run_id: str,
-        params: List[Param],
-        tags: List[RunTag],
-        metrics: List[Metric],
-        completion_event: threading.Event,
-    ) -> None:
+        run_id: Optional[str] = None,
```

Comment:
Q: Do we allow `None` for `run_id`?

Reply:
I used to have code that generates a dummy batch so that I made it optional, forgot to change back. Nice catch!